src/karmen_backend/tests/routes/test_gcodes.py

```diff
@@ -126,6 +127,26 @@ def test_limit(self):
                 in response.json["next"]
             )
 
+    def test_float_limit(self):
+        with app.test_client() as c:
+            c.set_cookie("localhost", "access_token_cookie", TOKEN_USER)
+            response = c.get(
+                "/organizations/%s/gcodes?limit=33.5&order_by=filename&fields=uuid,filename"
+                % UUID_ORG,
+                headers={"x-csrf-token": TOKEN_USER_CSRF},
```

Comment:
Actually no, but for some reason it's added to almost every request in tests. Looks like it is not necessary. Probably we should check why it was written with that since begging and then do some cleanup.

Reply:
I added new card about this issue to inbox

Reply:
It was used to retrieve gcode in FE as well so it seems the author just did not know details about CSRF so he just kept on safe side.
I have changed the card description slightly and added link to this dicussion.